platform-ui-skin/pom.xml

```diff
@@ -37,6 +37,7 @@
             <plugin>
                 <groupId>org.lesscss</groupId>
                 <artifactId>lesscss-maven-plugin</artifactId>
+                <version>1.7.0.1.1</version>

```

Comment:
I did a fair bit of trial and error re:
- Moving from Bootstrap 2.x to 3.x
- The lesscss maven plugin is no longer being maintained
- Less 2.x (not working with Matterhorn)
- what is the latest version of less that works with Java 1.8?

What worked was:

                <groupId>org.lesscss</groupId>
                <artifactId>lesscss-maven-plugin</artifactId>
                <version>1.7.0.1.1</version>

and (less 1.7.5):

                <lessJs>${project.basedir}/src/main/webapp/skin/third-party/less.js-1.7.5/dist/less-rhino-1.7.5.js</lessJs>

Which complied both bootstrap-2.3.2 and bootstrap-3.3.6

Now, you just need to migrate from bootstrap-2.3.2 to bootstrap-3.3.6 :)